rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -370,6 +372,12 @@ def test_upgrade_v4_v5(globaldb):
             key_parts=[GeneralCacheType.CURVE_POOL_TOKENS],
         )
         assert len(pool_tokens_in_global_db) == 0, 'All curve pool tokens should have been deleted'
+        expected_contracts = [
+            (contract[0], contract[1], contract[3], contract[4])  # remove name
+            for contract in contracts_before_upgrade
+        ]
```

Comment:
why not select everything except the name column and then here you don't have to do anything?